pkg/spxrpc/stubs.go

```diff
@@ -48,9 +49,27 @@ func (s *Server) RegisterHandlersComAtproto(e *echo.Echo) error {
 }
 
 func (s *Server) RegisterHandlersPlaceStream(e *echo.Echo) error {
+	e.GET("/xrpc/place.stream.graph.getFollowingUser", s.HandlePlaceStreamGraphGetFollowingUser)
 	return nil
 }
 
+func (s *Server) HandlePlaceStreamGraphGetFollowingUser(c echo.Context) error {
+	ctx, span := otel.Tracer("server").Start(c.Request().Context(), "HandlePlaceStreamGraphGetFollowingUser")
+	defer span.End()
+
+	userDID := c.Request().Header.Get("X-User-DID")
+	subjectDID := c.QueryParam("subjectDID")
+
+	var out *placestreamtypes.GraphGetFollowingUser_Output
+	var handleErr error
+	// func (s *Server) handlePlaceStreamGraphGetFollowingUser(ctx context.Context,subjectDID string) (*placestreamtypes.GraphGetFollowingUser_Output, error)
+	out, handleErr = s.handlePlaceStreamGraphGetFollowingUser(ctx, userDID, subjectDID)
+	if handleErr != nil {
+		return handleErr
+	}
+	return c.JSON(200, out)
+}
+
```

Comment:
this file gets autogenerated by `make lexicons`, should move it to a adjacent file